browser_patches/chromium/UPSTREAM_CONFIG.sh

```diff
@@ -0,0 +1,3 @@
+#      CURRENT_VERSION: 101.0.4951.15
```

Comment:
whats the reasoning for the extra spaces?

Reply:
just to align visually - looks beautiful!